File: python/raindrops/raindrops.py
factors = [(3, "Pling"), (5, "Plang"), (7, "Plong")]

def convert(number):
    result = [verb for factor, verb in factors if number % factor == 0]
    if not bool(len(result)): result.append(str(number))
    return ''.join(result)

File: go/raindrops/raindrops.go
package raindrops

import "strconv"

func Convert(n int) (ret string) {
  if n % 3 == 0 {
    ret += "Pling"
  }
  if n % 5 == 0 {
    ret += "Plang"
  }
  if n % 7 == 0 {
    ret += "Plong"
  }
  if len(ret) == 0 {
    ret = strconv.Itoa(n)
  }
  return
}
